### DISCUSSION TITLE
Icon font problem on Windows Server 2012 R2

### DISCUSSION THREAD

**Discussion by ray007** [cefsharp/CefSharp#4186] [Q&A]
I'm using CefSharp WPF to display a (local) webapp which uses the material icon font from google.
Our support now got a complaint from a customer, that there were no icons to see on his Windows server, instead the icon names were displayed everywhere.

Tracking back through my old versions, the problem seems to have appeared with an upgrade from CefSharp 75 to 85 back in 2020.
Looking at the same webapp with current google chrome, the icons are shown without any problems.

Any idea what's going wrong here and how I could maybe fix it?

**Comment by amaitland** [comment]
It's important to remember that CefSharp is just one of many chromium embedded framework wrappers. CEF has it's own support forum at https://magpcss.org/ceforum/index.php and issue tracker on bitbucket. 

Likely what you are seeing is https://bitbucket.org/chromiumembedded/cef/issues/2858/fonts-not-loading-look-wrong-on-windows-7

**Reply by ray007** [reply]
Thank you, I've added my comment there too.
